build_html.sh

```diff
@@ -24,7 +24,7 @@ do
         printf '/:version/*    /:version/en/:splat    301\n' >> build/html/_redirects
     else
         printf '/:version/%s/* /:version/%s/404.html  404\n' "$lang" "$lang" >> build/html/_redirects
-        printf '/:version/*    /:version/%s/:splat    301 Language=%s\n' "$lang" "${lang/_/-}" >> build/html/_redirects
+        printf '/:version/*    /:version/%s/:splat    301 Language=%s\n' "$lang" "`echo "$lang"  | sed 's/_/-/g'`" >> build/html/_redirects
```

Comment:
```suggestion
        printf '/:version/*    /:version/%s/:splat    301 Language=%s\n' "$lang" "$(printf "%s" "$lang"  | sed 's/_/-/g')" >> build/html/_redirects
```
Printf is a shell builtin (in contrast to echo) and `$()` is more readable than legacy backtick notstion IMHO.